spec/spec_helper.rb

```diff
@@ -69,6 +69,10 @@
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
 
+  # TODO - remove this line after Spree upgrade. Current Spree specs do not have `type: :controller` definitions.
+  # This is a workaround to pass tests in Travis i.e. https://travis-ci.org/openfoodfoundation/openfoodnetwork/jobs/235601640
+  config.infer_spec_type_from_file_location!
+
```

Comment:
Mainly that it is not required. I don't think it is the reason you are seeing the failure you are seeing. We are setting `ActionMailer::Base.default_url_options[:host]` and `Rails.application.routes.default_url_options[:host]` in a bunch of different specs, my guess would be that the spec that is failing has jumped from one thread to another on CI and is preceded by a different set of specs than before and is so being executed in a context where the default host has changed... Another case of tech debt that hasn't been addressed...